oaknational/Oak-Web-Application
# Extract out unit card used in curriculum options

## Description
Fixes issue where wobbly borders still appeared on unit options

## Issue(s)

Fixes `CUR-1028`

## How to test

1. Go to https://deploy-preview-2960--oak-web-application.netlify.thenational.academy
2. Navigate to curriculum visualiser
3. Check unit options doesn't have wobbly borders
4. Check for regressions 

## Screenshots
Before

<img width="386" alt="Screenshot 2024-11-05 at 18 32 13" src="https://github.com/user-attachments/assets/b8203858-033e-4a18-8b23-0fbab6a85d18">

After

<img width="386" alt="Screenshot 2024-11-05 at 18 31 36" src="https://github.com/user-attachments/assets/cac5135e-c4c7-4723-bafb-6d3d693a7a93">
